lib/data-helper.js

```diff
@@ -253,3 +253,29 @@ export function getCalypsoLiveQuery() {
 	}
 	return 'branch=' + config.get( 'branchName' );
 }
+
+export function getCalypsoURL( route = '', queryStrings = [] ) {
+	let queryString = '';
+	let url = this.configGet( 'calypsoBaseURL' ) + '/' + route;
+
+	for ( let qs of queryStrings ) {
+		queryString = this._appendQueryString( queryString, qs );
+	}
+
+	if ( this.isRunningOnLiveBranch() ) {
+		if ( url.indexOf( 'calypso.live' ) < 0 ) {
```

Comment:
This isn't really necessary but I thought it would be nice to stop any accidental errors on trying live branch testing using a different environment like wordpress.com (sometimes I forget to update the config URL when debugging)

Reply:
Makes sense to me!